protocol/avdl/keybase1/tlf_keys.avdl

```diff
@@ -4,6 +4,12 @@
 // getting TLF keys
 protocol tlfKeys {
   import idl "common.avdl";
+  import idl "identify.avdl";
+
+  enum TLFIdentifyBehavior {
+    DEFAULT_KBFS_1,
+    CHAT_2
```

Comment:
Sounds good; maybe also a `CHAT_CLI` which is essentially the same behavior as `DEFAULT_KBFS` but just for clarity?